v3-nginx/docker-entrypoint.d/90-nginx-extras-envsubst.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+# vim:sw=2:ts=2:sts=2:et
+
+set -eu
+
+LC_ALL=C
+ME=$( basename "$0" )
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+
+export DNS_SERVER="${DNS_SERVER:-$(grep -i '^nameserver' /etc/resolv.conf |head -n1|cut -d ' ' -f2)}"
+
+touch /etc/nginx/nginx.conf 2>/dev/null || { echo >&2 "$ME: error: can not modify /etc/nginx/nginx.conf (read-only file system?)"; exit 0; }
```

Comment:
```suggestion
touch /etc/nginx/nginx.conf 2>/dev/null || { echo >&2 "$ME: ERROR: can not modify /etc/nginx/nginx.conf (read-only file system?)"; exit 0; }
```

Reply:
This came from upstream, but removed in new iteration.